static/js/publisher/metrics/metrics.js

```diff
@@ -34,4 +38,16 @@ function renderMetrics(metrics) {
   territoriesMetrics(metrics.territories.selector, metrics.territories.metrics);
 }
 
-export default renderMetrics;
+/**
+ * Render publisher page metrics
+ * @param {Object} options An object with rendering options.
+ */
+function renderPublisherMetrics(options) {
```

Comment:
Name of the `renderPublisherMetrics` function seems to be more specific than `renderMetrics`, but they both practically render publisher metrics... just different ones.

As this are metrics for 'My snaps' page, maybe it could be called `renderMySnapsMetrics` or (because it renders snap installs) just `renderSnapInstalls`?